### PR TITLE
Update text for notebook runtime environment change warning [SATURN-1047]

### PR DESCRIPTION
**Pull request by ehanna4**
Update the text warning the user of the effect of updating a Notebook Runtime environment to more accurately reflect actual behavior.

### CODE REVIEW

File: src/components/ClusterManager.js [hunk 1]
```diff
@@ -307,8 +307,10 @@ export class NewClusterModal extends PureComponent {
       ]),
       changed ?
         div({ style: styles.warningBox }, [
-          'Updated environments take a few minutes to prepare. ',
-          'You will be notified when it is ready and can continue working as it builds.'
+          div({ style: styles.label }, ['Caution:']),
+          div({ }, ['Updating a Notebook Runtime environment will delete all existing non-notebook files and ' +
```
Comment (panentheos) [inline src/components/ClusterManager.js]: nit: If the props object is empty, you can omit it entirely.

File: src/components/ClusterManager.js [hunk 2]
```diff
@@ -307,8 +307,10 @@ export class NewClusterModal extends PureComponent {
...
+          'installed packages. You will be unable to work on the notebooks in this workspace while it ' +
+          'updates, which can take a few minutes.'])
```
Comment (panentheos) [inline src/components/ClusterManager.js]: code style nit: if the children array spans multiple lines, we usually expand it out fully and indent like this. (Note the spaces at the end of the strings to make sure the text doesn't run together in this case)
```
div([
  'blah ',
  'blah ',
  'blah'
])
```

Comment (ehanna4) [reply src/components/ClusterManager.js]: @panentheos Thank you for the feedback! I updated the code per your recommendations.

**Review by panentheos — APPROVED** [review]
Looks good, thanks!

**Pull request merged**